### DISCUSSION TITLE
httpx -cname problem?

### DISCUSSION THREAD

**Discussion by Phoenix1112** [projectdiscovery/httpx#1706] [Q&A]
I have many subdomain addresses with cname addresses, but when I use the ```httpx -cname``` command, it finds the cname of only one subdomain, but it ends without showing the others?

**Comment by ehsandeep** [comment]
@Phoenix1112 please always include example that can be used to review / reproduce.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
